### DISCUSSION TITLE
Problems to be improved（video）

### DISCUSSION THREAD

**Discussion by optical-core** [opentoonz/opentoonz#6403] [Ideas]
### Is your feature request related to a problem? Please describe

https://user-images.githubusercontent.com/39340555/216247625-d96f9750-5ff7-4406-ba51-dde841cc1756.mp4

Q1. Timesheet Step +1 -1
Q2. insert drawing & delete drawing.
Q3. space key move stage mode.
Q4. selection tool subtract selections.
Q5. curve tool mouse trigger mode.
Q6. Export vector Level rendering as binarized pictures.
Q7. Betch vector level palette order automatic sorting.

### Describe the solution you'd like

Q1. Timesheet Step +1 -1 has select only one step.
Q2. delete drawing keep number unchanged in time.
![image](https://user-images.githubusercontent.com/39340555/216253785-6d8493a6-75b6-45d5-93ac-9dc323b403ea.png)

Q3. space key move stage mode does not affect selection.（Reference middle click）
Q4. selection tool hold Alt to subtract selections.
Q5. curve tool mouse trigger mode with only two steps.
Q6. Export vector Level rendering as binarized pictures.
Q7. Betch vector level palette order automatic sorting.

**Comment by flurick** [comment]
Q1 Fixed in https://github.com/tahoma2d/tahoma2d/commit/0335bda0536cb0a24c4ff6fb9f0e40352ccbf7c9
Q3 discussion and fix https://github.com/tahoma2d/tahoma2d/pull/588--

**Comment by RodneyBaker** [comment]
Converting this to Discussion so others can explore information and consider the best way forward for areas mentioned some of which have been implemented, some implemented in Tahoma2D but not yet ported, etc.

**Comment by Abel032** [comment]
我尽量在1.8之前完成这些
I think I can implement these before the 1.8 release.
